lib/trade_tariff_admin.rb

```diff
@@ -18,7 +18,15 @@ def production?
     end
 
     def authenticate_with_sso?
-      ENV.fetch('AUTHENTICATE_WITH_SSO', 'true') == 'true'
+      @authenticate_with_sso ||= ENV.fetch('AUTHENTICATE_WITH_SSO', 'true') == 'true'
+    end
+
+    def basic_session_authentication?
+      @basic_session_authentication ||= !authenticate_with_sso? && basic_session_password.present?
```

Comment:
Is there an argument to always enable basic session auth when sso is disabled?

Reply:
We can disable both in local development